packages/app/app/containers/SoundContainer/index.js

```diff
@@ -167,11 +168,24 @@ class SoundContainer extends React.Component {
     );
   }
 
+  isHlsStream(url) {
+    return /http.*?\.m3u8/g.test(url);
+  }
+
   render() {
     const { queue, player, equalizer, actions, enableSpectrum, currentStream, location, defaultEqualizer } = this.props;
     const currentTrack = queue.queueItems[queue.currentSong];
     const usedEqualizer = enableSpectrum ? equalizer : defaultEqualizer;
-    return Boolean(currentStream) && (
+    return Boolean(currentStream) && (this.isHlsStream(currentStream.stream) ? (
+      <HlsPlayer 
```

Comment:
Will this work with the visualizer?

Reply:
currently it doesn't.

Reply:
Ok, maybe in the future we could think about streaming the sound through the same pipeline as everything else